libs/sdk-ui-kit/src/List/InvertableList.tsx

```diff
@@ -6,12 +6,12 @@ import keyBy from "lodash/keyBy";
 import values from "lodash/values";
 import take from "lodash/take";
 import has from "lodash/has";
+import { Dictionary } from "lodash";
 import { guidFor } from "@gooddata/goodstrap/lib/core/Guid";
-import Message from "@gooddata/goodstrap/lib/Messages/Message";
 
-import { MultiSelectList } from "./MultiSelectList";
+import { Message } from "../Messages";
 import { Input } from "../Form";
-import { Dictionary } from "lodash";
+import { MultiSelectList } from "./MultiSelectList";
```

Comment:
I am surprised that it works. It should import LegacyMultiSelectList (the original one from GS from root folder, not from @next folder)

Reply:
Just a diff misinterpretation.